contents/blog/how-to-run-transparent-company.md

```diff
@@ -0,0 +1,135 @@
+---
+date: 2022-07-26
+title: "How to run a transparent startup"
+rootPage: /blog
+sidebar: Blog
+showTitle: true
+hideAnchor: true
+categories: ["CEO diaries", "Inside PostHog"]
+author: ["james-hawkins"]
+featuredImage: ../images/blog/posthog-ceo-diary-blog.png
+featuredImageType: full
+---
+
+There really hasn't been a single situation where sharing pretty much everything we do hasn't been to our company's advantage.
+
+Most companies seem to share when necessary but not by default. I hope this piece changes that - as a startup, you've done something wrong if you don't trust your own team.
+
+Being realistic, to help this most people - this is mainly written about _internal_ transparency... and it's startup-focussed too, since that's what I know.
```

Comment:
I don't understand some of how this is expressed. I think I'd probably reword this to something like:

```suggestion
It's worth noting that these are my views on how and why more startups should offer greater _internal_ transparency. I've focused on internal transparency because I believe it can be realistically practiced and helps the most people. I've written about startups because that's what I know.
```

It's wordier, I know. :(